doc/example-common.yaml

```diff
@@ -26,30 +26,33 @@ jupyterhub:
   hub:
     extraEnv:
       OAUTH2_AUTHORIZE_URL: "https://auth.mast.stsci.edu/oauth/authorize"
-      OAUTH_CALLBACK_URL: "https://roman-sit.science.stsci.edu/hub/oauth_callback"
+      # OAUTH2_AUTHORIZE_URL: "https://auth.masttest.stsci.edu/oauth/authorize"
```

Comment:
Should we have the actual URL here? Could make it a target

Reply:
Same with url's below

Reply:
Can you clarify please?  Not sure what you mean.

Reply:
https://roman-sit.science.stsci.edu/hub/oauth_callback url - by showing this here it could make it a target for a DDOS or some sort of attack.  Not sure if I'm being overly cautious

Reply:
Hmm.  That is a good point.  Although, that URL is useless if the credentials are unknown.

Reply:
I would think DDOS attacks are MAST's responsibility to handle.